segmentify/main.py

```diff
@@ -20,13 +20,19 @@ def main(args):
         imgs = np.array(imgs)
 
     with gui_qt():
-        viewer = Viewer(imgs, heatmap=args.heatmap)
-
-
-if __name__ == "__main__":
+        try:
```

Comment:
The simple examples in the README and examples folder don't use heatmaps, and expecting but not getting one was cuasing an error. 
I'm not quite sure how to pass in a heatmap using command line arguments, but I assume this part will work if you do?